recipes/opencensus-ext-azure/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "opencensus-ext-azure" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 39f3ad44b2ca7b1f35ea13729f9cf5cb5db8f96ea1ff4a68caf9c15b26249334
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - opencensus >=0.8.dev0, <1.0.0
+    - psutil >=5.6.3
+    - requests >=2.19.0
```

We can use the deps at the specific commit tagged `v0.7.11`:
https://github.com/census-instrumentation/opencensus-python/blob/v0.7.11/contrib/opencensus-ext-azure/setup.py#L41-L45
```suggestion
    - python
    - opencensus >=0.7.11,<1.0.0
    - psutil >=5.6.3
    - requests >=2.19.0
```